source/base/Text.ooc

```diff
@@ -53,6 +53,30 @@ Text: cover {
 	operator != (other: String) -> Bool { !(this == other) }
 	operator != (other: This) -> Bool { !(this == other) }
 	operator + (other: This) -> This { This new(this _buffer + other _buffer) }
+	operator + (other: Int) -> This {
+		converted := other toString()
```

Comment:
Right, but if `Text` was in the SDK, you wouldn't need to `use` anything, as is the case with `String`. For what that's worth.